bench/README.md

```diff
@@ -18,40 +28,62 @@ pip3 install -r bench/requirements.txt
+  * Each write transactions is signed and thus requires:
 
-* an account with write permissions (e.g. Vendor account)
-* proper values for txn `sequence` which enforces txns ordering for an account
+    * an account with write permissions (e.g. Vendor account)
+    * proper values for txn `sequence` which enforces txns ordering for an account
 
-By that reason load test uses prepared load data which can be generated as follows:
+  * Initialize the pool and test accounts (**Warning** applicable to local in-docker pool only for now):
 
-* Initialize the pool and test accounts (**Warning** applicable to local in-docker pool only for now):
+      ```bash
+      make localnet_clean
 
-    ```bash
-    make localnet_clean
+      make localnet_init
 
-    # DCL_OBSERVERS=1 make localnet_init  # to initialize observers as well
-    make localnet_init
-    
-    # ./gentestaccounts.sh [<NUM-USERS>]
-    ./gentestaccounts.sh
+      make localnet_start
+      # Note: once started ledger may require some time to complete the initialization.
+      ```
 
-    make localnet_start
-    # Note: once started ledger may require some time to complete the initialization.
-    ```
-
-* Generate test transactions:
+  * Copy local accounts keys to folder `~/.dcl/keyring-test`:
+      ```bash
+      cp ./.localnet/node0/keyring-test/* ~/.dcl/keyring-test
+      ```
 
+* Copy accounts keys to folder `~/.dcl/keyring-test` (**this item does not need for the local pool**):
+    * After connecting to the network, we must copy the `Trustee` account keys to `~/.dcl/keyring-test`. <br>
+    `The number of accounts with role Trustee must not exceed 3`.
+* Enter to `bench` folder
     ```bash
-    # DCLBENCH_WRITE_USERS_COUNT=<NUM-USERS> DCLBENCH_WRITE_USERS_Q_COUNT=<NUM-REQ-PER-USER> DCLBENCH_ADD_NEW_MODELS_COUNT=<NUM-NEW-MODELS> python bench/generate.py bench/test.spec.yaml.j2 bench/txns
-    python bench/generate.py bench/test.spec.yaml.j2 bench/txns
+    cd bench
     ```
 
-Here the following (**optional**) inputs are considered:
-
-* `NUM-USERS`: number of client accounts with write access (created as Vendors). Default: 10
-* `NUM-REQ-PER-USER`: number of write txns to perform per a user. Default: 1000
-* `NUM-NEW-MODELS`: models are added to the ledger for reading afterwards in load tests. Default: 5
+* Open the .env file:
+    * update fields.<br>
+        e.g.:
+        ```yml
+        DCLD_VERSION=v0.11.0
+        DCLD_NODE=tcp://host.docker.internal:26657
+        DCLD_CHAIN_ID=dclchain
+
+        WRITE_HOSTS=http://host.docker.internal:26657
+        READ_HOSTS=http://host.docker.internal:26640
+        TRUSTEE_ACCOUNT_NAME=jack
+        COUNT_USERS=4 
+        ```
+        `<DCLD_VERSION>` - dcld binary version.<br>
+        `<DCLD_NODE>` - Address `<host>:<port>` of the node to connect. This node needs for adding account with role `Vendor` in write load tests.<br>
```

Comment:
Please explicitly mention (for DCLD_NODE, WRITE_HOSTS, READ_HOSTS) that it can either point to a local in-docker nodes, or to external nodes. Please mention what value should be used for the local in-docker case.

Reply:
Done! ✅